app/code/Magento/Eav/Test/Unit/Model/AttributeSetRepositoryTest.php

```diff
@@ -212,7 +212,7 @@ public function testGetList()
         $searchCriteriaMock = $this->getMock('\Magento\Framework\Api\SearchCriteriaInterface');
 
         $filterGroupMock = $this->getMock('\Magento\Framework\Api\Search\FilterGroup', [], [], '', false);
-        $searchCriteriaMock->expects($this->once())->method('getFilterGroups')->willReturn([$filterGroupMock]);
+        $searchCriteriaMock->expects($this->exactly(2))->method('getFilterGroups')->willReturn([$filterGroupMock]);
 
         $filterMock = $this->getMock('\Magento\Framework\Api\Filter', [], [], '', false);
         $filterGroupMock->expects($this->once())->method('getFilters')->willReturn([$filterMock]);
```

Comment:
@davidverholen I think this is the method that needs to be called more than once in the tests.